model.md

```diff
@@ -0,0 +1,66 @@
+# Pedigree conceptual model
+
+## Motivation
+
+Lack of interoperability of pedigree data between systems results in a substantial double-entry burden for already-overloaded nurses, genetic counselors, and support staff, and introduces the potential for serious errors.
+
+# Classes
+
+![image](https://user-images.githubusercontent.com/4251264/105409205-1c53b200-5bfe-11eb-9d77-ed5630138eaf.png)
+
+## Pedigree
+
+A `Pedigree` is collection of selected information about a family, including the individuals, relationships between them, and relevant health conditions.
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| proband | ID | optional | id of Individual that is the index case for the family |
+| consultand | ID | optional | id of Individual that is the focus of the current analysis |
+| collectedAt | Date | optional | the date the pedigree was collected, as ISO full or partial date, i.e. YYYY, YYYY-MM, or YYYY-MM-DD |
+| reasonCollected | Concept | optional | the health condition being investigated in the family; if any Individual has the `affected` property defined, it refers to this condition |
+
+## Individual
+
+### properties
+
+| Field | Type | Status | Definition |
+| --- | --- | --- | --- |
+| id | ID | required | logical id |
+| name | string | optional | name of the individual |
+| identifiers | list of string | optional | external identifiers for the individual |
+| sex | Concept | optional | Sex for Clinical Use, values: `male`, `female`, `other`, `unknown`; See: https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions and https://www.hl7.org/fhir/valueset-administrative-gender.html |
+| gender | Concept | optional | Presumed or reported gender identity, values: `male`, `female`, `non-binary`, `non-disclosed`, `trans`, ...; See: https://confluence.hl7.org/display/VOC/Gender+Harmony+Context+Definitions |
```

Comment:
This is from the current preview (3) of the upcoming FHIR release (R5): https://build.fhir.org/patient.html#gender

I fully understand that this modeling effort doesn't rely on any specific CDMs. I am totally down for the idea of separating administrative gender from clinical (or karyotypic) sex as suggested above. However, 1) the above model seems to swap the concepts of sex and gender (considering against FHIR) and 2) whether we want to keep both attributes at the same time at Individual seems worth a discussion.

Reply:
Good point, and thank you for sharing that link. It's helpful to see the FHIR perspective summarized in one place. I see the confusion, and am not sure the best approach for our use case. Let's discuss on the call today.